components/stepper/index.css

```
@@ -535,21 +511,24 @@ governing permissions and limitations under the License.
 
 .spectrum-Stepper-stepUp {
   padding-block-start: var(--mod-stepper-icon-nudge-start, var(--spectrum-stepper-icon-nudge-start));
+  padding-block-end: 0;
 
-  border-start-start-radius: var(--mod-stepper-button-border-radius, var(--spectrum-stepper-button-border-radius));
-  border-start-end-radius: var(--mod-stepper-button-border-radius, var(--spectrum-stepper-button-border-radius));
-  border-end-start-radius: 0;
-  border-end-end-radius: 0;
+  border-start-start-radius: var(--mod-stepper-button-border-radius-reset, var(--spectrum-stepper-button-border-radius-reset));
+  border-start-end-radius: var(--mod-stepper-border-radius, var(--spectrum-stepper-border-radius));
+  border-end-start-radius: var(--spectrum-stepper-button-border-radius-reset);
+  border-end-end-radius: var(--spectrum-stepper-button-border-radius-reset);
+  border-bottom-width: var(--spectrum-stepper-button-border-radius-reset);
 }
 
 .spectrum-Stepper-stepDown {
   border-block-start-width: var(--mod-stepper-button-middle-border-width, var(--spectrum-stepper-button-middle-border-width));
   padding-block-end: var(--mod-stepper-icon-nudge-end, var(--spectrum-stepper-icon-nudge-end));
+  padding-block-start: 0;
 
-  border-start-start-radius: 0;
-  border-start-end-radius: 0;
-  border-end-start-radius: var(--mod-stepper-button-border-radius, var(--spectrum-stepper-button-border-radius));
-  border-end-end-radius: var(--mod-stepper-button-border-radius, var(--spectrum-stepper-button-border-radius));
+  border-start-start-radius: var(--spectrum-stepper-button-border-radius-reset);
+  border-start-end-radius: var(--spectrum-stepper-button-border-radius-reset);
+  border-end-start-radius: var(--mod-stepper-button-border-radius-reset, var(--spectrum-stepper-button-border-radius-reset));
```

Let's just drop the `--mod-` here. It looks like this is the only place we have it defined, and due to the temporary nature of these changes, I'd rather exclude this for now so that it doesn't require future breaking changes or support.

```suggestion
  border-end-start-radius: var(--spectrum-stepper-button-border-radius-reset);
```

---

On it!

---

Ok this has been updated and the mods file regenerated